From 2193a9800b1d6ae771539e1cdae1e7340b9d90ea Mon Sep 17 00:00:00 2001
From: Joan Jani <igiann@hotmail.com>
Date: Thu, 6 Jul 2017 20:35:32 +0000
Subject: [PATCH] Bluetooth: btqca: Fixed a coding style error
Git-commit: 2193a9800b1d6ae771539e1cdae1e7340b9d90ea
References: git-fixes
Patch-mainline: v4.14-rc1

Fixed this coding style erro

./drivers/bluetooth/btqca.c:84: ERROR: code indent should use tabs where possible

Signed-off-by: Joan Jani <igiann@hotmail.com>
Signed-off-by: Marcel Holtmann <marcel@holtmann.org>
Signed-off-by: Oliver Neukum <oneukum@suse.com>

---
 drivers/bluetooth/btqca.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bluetooth/btqca.c b/drivers/bluetooth/btqca.c
index 28afd5d585f9..0bbdfcef2aa8 100644
--- a/drivers/bluetooth/btqca.c
+++ b/drivers/bluetooth/btqca.c
@@ -81,7 +81,7 @@ static int rome_patch_ver_req(struct hci_dev *hdev, u32 *rome_version)
 	 * and lower 2 bytes from patch will be used.
 	 */
 	*rome_version = (le32_to_cpu(ver->soc_id) << 16) |
-		        (le16_to_cpu(ver->rome_ver) & 0x0000ffff);
+			(le16_to_cpu(ver->rome_ver) & 0x0000ffff);
 
 out:
 	kfree_skb(skb);
-- 
2.44.0

